File: go/1041.go
// Funcional no beecrowd
package main

import (
	"fmt"
)

func main() {
	var X float64
	var Y float64

	fmt.Scan(&X)
	fmt.Scan(&Y)

	if X == 0 && 0 != Y {
		fmt.Println("Eixo Y")
	} else if X != 0 && Y == 0 {
		fmt.Println("Eixo X")
	} else if X > 0 && Y > 0 {
		fmt.Println("Q1")
	} else if X < 0 && Y > 0 {
		fmt.Println("Q2")
	} else if X < 0 && Y < 0 {
		fmt.Println("Q3")
	} else if X > 0 && Y < 0 {
		fmt.Println("Q4")
	} else {
		fmt.Println("Origem")
	}
}

File: Lua/Iniciante/1041.lua
-- Funcional no beecrowd!
local x = io.read("n")
local y = io.read("n")

if x == 0 and 0 ~= y then
    print("Eixo Y")
elseif y == 0 and 0 ~= x then
    print("Eixo X")
elseif x > 0 and 0 < y then
    print("Q1")
elseif x < 0 and 0 < y then
    print("Q2")
elseif x < 0 and 0 > y then
    print("Q3")
elseif x > 0 and 0 > y then
    print("Q4")
else
    print("Origem")
end